AdminWebsite/AdminWebsite/ClientApp/src/app/services/justice-users.service.ts

```diff
@@ -11,15 +11,18 @@ export class JusticeUsersService {
     private cache$: Observable<JusticeUserResponse[]>;
 
     constructor(private apiClient: BHClient) {}
-
-    retrieveJusticeUserAccounts(term: string) {
+    retrieveJusticeUserAccounts() {
         if (!this.cache$) {
-            this.cache$ = this.requestJusticeUsers(cleanQuery(term)).pipe(shareReplay(1));
+            this.cache$ = this.requestJusticeUsers(cleanQuery(null)).pipe(shareReplay(1));
         }
 
         return this.cache$;
     }
 
+    retrieveJusticeUserAccountsNoCache(term: string) {
```

Comment:
could we add a test for this in justice-users.service.spec.ts?

Reply:
added